tests/data/test1568

```diff
@@ -0,0 +1,87 @@
+<testcase>
+# based on test 64
+
+<info>
+<keywords>
+HTTP
+HTTP GET
+HTTP Digest auth
+</keywords>
+</info>
+
+<reply>
+<data>
+HTTP/1.1 401 Authorization Required swsclose
+Server: Apache/1.3.27 (Darwin) PHP/4.1.2
+WWW-Authenticate: Digest realm="testrealm", nonce="1053604145"
+Content-Type: text/html; charset=iso-8859-1
+Content-Length: 26
+
+This is not the real page
+</data>
+
+# This is supposed to be returned when the server gets a
+# Authorization: Digest line passed-in from the client
+<data1000>
```

Comment:
Why `data1000`?

Reply:
Because the sws test server automatically adds 1000 to the "part number" when it finds `Authorization: Digest` in the request as way for it to provide a different response with and without that header present. I see now that this is not documented in `FILEFORMAT.md`, so I'll create a separate PR to try to mention some of the part number magic the server does based on contents in the request.

Reply:
Oh, never mind, it *is* mentioned in that document already!